pymc3/distributions/arimastan.py

```diff
@@ -0,0 +1,289 @@
+import numpy as np
+
+import pymc3 as pm
+
+
+def invchi(x, chisquare):
+    return (1.0 / x ** 2) * chisquare.logp(1 / x)
+
+
+class Sarima:
```

Comment:
SARIMA, to be consistent with AR, GARCH, etc.

Reply:
will do, thanks!